ihs/src/main/scripts/was-check.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+#      Copyright (c) Microsoft Corporation.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+# 
+#           http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
```

Comment:
Why not have the set and IFS here?

Reply:
That's because this script has its own logic to handle different situations, e.g., writing the result to the log file, cleaning up in the end, instead of just exit in case of failure of command execution in the script.